src/m365/pp/commands/environment/environment-get.ts

```diff
@@ -0,0 +1,81 @@
+import { Logger } from '../../../../cli/Logger';
+import { AxiosRequestConfig } from 'axios';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import PowerPlatformCommand from '../../../base/PowerPlatformCommand';
+import { Environment } from '../Environment';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  name: string;
+  asAdmin?: boolean;
+}
+
+class PpEnvironmentGetCommand extends PowerPlatformCommand {
```

Comment:
do you think we could add `defaultProperties()` ? 
I know it's a bit uncommon in `get` commands but it is used where the `text` output will be unreadable at all. For example we have it in `flow get` command as well. 
so now if we do `--output text` we get 
![image](https://user-images.githubusercontent.com/58668583/194669221-4ebd1a7b-e41b-4916-8f30-27a53b786d5b.png)

but if we add
```
public defaultProperties(): string[] | undefined {
    return ['name', 'id'];
  }

```

the output will be a lot cleaner and more useful 
![image](https://user-images.githubusercontent.com/58668583/194669317-23f4e9e0-44ec-4bf9-a47e-aa9caaf8186e.png)


@appieschot also you are the spec author maybe any comments on this one?

Reply:
@Adam-it Not a bad idea, I did not know we had the `defaultProperties()` on other commands besides the list. In thise case I would align them with #3061 where we use `name, displayName` (the ID is not required as that matches the `name` property and is quite long.